pxf/pxf-service/src/main/resources/pxf-profiles-default.xml

```diff
@@ -101,6 +101,17 @@ under the License.
             <outputFormat>org.apache.hawq.pxf.service.io.GPDBWritable</outputFormat>
         </plugins>
     </profile>
+        <profile>
+        <name>HiveVectorizedORC</name>
```

Comment:
seems like "batch" and "vectorized" are used interchangeably, should we use just one term ?